app/src/main/java/io/intrepid/contest/screens/contestcreation/categorieslist/CategoriesListContract.java

```diff
@@ -9,18 +9,16 @@
 
     interface View extends BaseContract.View {
         void showCategories(List<Category> categories);
-        void showDefaultCategory();
         void showAddCategoryScreen();
         void showNextScreen();
         void showEditCategoryPage(Category category);
-        void onCategoryClicked(Category category);
+
+        Category getDefaultCategory(int categoryName, int categoryDescription);
```

Comment:
make sure to keep consistent formatting, either everything should not have spaces or there should be spaces between all the functions.